### PR TITLE
added tags for releasegroupid

### PR DESCRIPTION
**Pull request by zductiv**
Added MusicBrainzReleaseGroupId tag, tests for #98

### CODE REVIEW

File: src/TagLib/Asf/Tag.cs [hunk 1]
```diff
@@ -1201,23 +1201,43 @@ public override string Description
...
+        ///    A <see cref="string" /> containing the MusicBrainz 
+        ///    ReleaseID for the media described by the current
+        ///    instance or null if no value is present.
+        /// </value>
+        /// <remarks>
+        ///    This property is implemented using the "MusicBrainz/Album Id"
+        ///    field.
+        ///    http://musicbrainz.org/doc/PicardTagMapping
+        /// </remarks>
+        public override string MusicBrainzReleaseId {
```
Comment (decriptor) [inline src/TagLib/Asf/Tag.cs]: Can you make sure you don't convert tabs to spaces.

**Comment by decriptor** [comment]
In general it seems that you have converted tabs to space while adding code. Can I have you leave them as tabs.

**Comment by zductiv** [comment]
Looks like Visual Studio was being "helpful" when I was copy-pasting code. New to this so I went through the files and tabified them back.

File: src/TagLib/Asf/Tag.cs [hunk 2]
```diff
@@ -229,8 +229,8 @@ public string GetDescriptorString (params string [] names)
 		///    The value will be stored in the first value in <paramref
 		///    name="names" /> and the rest will be cleared.
 		/// </remarks>
-		public void SetDescriptorString (string value,
-		                                 params string [] names)
+		public void SetDescriptorString (string value,
+										 params string [] names)
```
Comment (decriptor) [inline src/TagLib/Asf/Tag.cs]: Sorry, another formatting issue. It should have tabs up to the beginning of the method and then spaces to align the parameter under the first. I think the IDE is trying to be too helpful again :(

If you could fix this that'd be great. It help reduce the size of the code diff and as a result makes review the PR much easier.

File: src/TagLib/CombinedTag.cs [hunk 3]
```diff
@@ -1270,8 +1270,50 @@ public override string[] PerformersRole
...
+		///    <para>When setting the value, it is stored in each child
+		///    tag.</para>
+		/// </remarks>
+		/// <seealso cref="Tag.MusicBrainzReleaseGroupId" />
+		public override string MusicBrainzReleaseGroupId
+		{
+			get
+			{
+				foreach (Tag tag in tags)
+				{
```
Comment (decriptor) [inline src/TagLib/CombinedTag.cs]: I think this is the last nitpick about formatting. The opening curly brace should be on the same line. ie,
`
		public override string MusicBrainzReleaseGroupId {
			get {
				foreach (Tag tag in tags) {
`

File: src/TagLib/Ogg/GroupedComment.cs [hunk 4]
```diff
@@ -1378,22 +1414,22 @@ public override string[] PerformersRole
 	/// <seealso cref="Tag.ReplayGainTrackGain" />
 	public override double ReplayGainTrackGain
 	{
-  	get
-  	{
+	get
```
Comment (decriptor) [inline src/TagLib/Ogg/GroupedComment.cs]: This is a good change, but it sadly pollutes the PR. I wouldn't worry about fixing this one though.

**Comment by majora2007** [comment]
@decriptor Can we merge this PR?

**Comment by decriptor** [comment]
@majora2007 sorry about the delay. I'll see about getting this merged by the end of the week.

**Comment by decriptor** [comment]
I'll try and fix a few of the last formatting issues and then merge this tonight.

**Comment by Qstick** [comment]
@decriptor anything new on this PR?

**Comment by decriptor** [comment]
Sorry this took forever

**Pull request merged**